modules/ocf_hpc/facts.d/nvidia_gpu_info

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import json
+import re
+import shutil
+import subprocess
+
+
+def main():
+    if not shutil.which('nvidia-smi'):
```

Comment:
Wouldn't the check below for `subprocess.CalledProcessError` handle this? In any case, I think it's probably better to keep this check here and remove the exception handling, since failing loudly on that is probably good.

Reply:
might be worth outputting "are you sure you are on an HPC machine?"